channels/candidate-4.2.yaml

```diff
@@ -30,3 +31,4 @@ versions:
 - 4.2.12-s390x
 - 4.2.13
 - 4.2.13-s390x
+- 4.2.14
```

Comment:
oh, wait, because both the s390x and amd64 images share this name, we want `4.2.14+amd64` here to only promote the amd64 release.  Or 4.2.14 to promote both amd64 and s390x.  I'll work up a patch to the pusher script to handle the optional build information.

Reply:
cool, that was gonna be my question... what if we decide not to promote one (though frankly that would be a mess for other reasons... we can't separate out the releases in errata-tool or in the appregistry)

Reply:
#16 has my WIP in this direction (just the README, but folks can discuss my approach there while I work up the implementation).

Reply:
and this PR is good for promoting both as it stands (assuming my approach holds up ;).

And you could promote both to candidate channels but then only promote amd64 to fast/stable if, for example, s390x turned out to be unstable.  Does that address the errata concern?